src/routes/days.js

```diff
@@ -1,11 +1,10 @@
 let router = require("express").Router();
 let { bookDaysOff } = require("../actions/bookingDaysOffActions");
 let { deleteDaysOff } = require('../actions/deleteDaysOffActions')
-let { getDaysOff } = require('../actions/getNumberOfDaysOffActions')
+let { getNumberOfDaysOff } = require('../actions/getNumberOfDaysOffActions')
 let { retriveAlreadyBookedDays } = require('../actions/retriveAlreadyBookedDaysActions')
 
 router.post("/", bookDaysOff);
 router.delete('/:id', deleteDaysOff)
-router.get('/:id', getDaysOff)
-
+router.get('/:id/getNumberOfDaysOff', getNumberOfDaysOff)
```

Comment:
I could switch the route from '/days' to '/users' . I think this is the best solution because this way we make only 2 searches in the database. Other solution would be the obtaining user's id after searching by id of daysOff's resource. This way we would do one more search.
It would look like this: `/users/getNumberOfDaysOff/:id`
 Which solution shall I choose?

Reply:
`/users/getNumberOfDaysOff/:id` doesn't sound very restful and it seems that was the chosen api style.

Reply:
How does `/users/:id/available-days-off` sound to you?